### PR TITLE
BXMSDOC-4998 Optaplanner: Update headings to title case

### PR DESCRIPTION
**Pull request by emmurphy1**

### CODE REVIEW

File: optaplanner-docs/src/main/asciidoc/BenchmarkingAndTweaking/BenchmarkingAndTweaking-chapter.adoc [hunk 1]
```diff
@@ -169,11 +169,11 @@ but instead are added to the tail of the solver phases list.
 
 
 [[solutionFileIO]]
-=== SolutionFileIO: Input And Output Of Solution Files
+=== `SolutionFileIO`: input and output of solution files
```
Comment (ge0ffrey) [inline optaplanner-docs/src/main/asciidoc/BenchmarkingAndTweaking/BenchmarkingAndTweaking-chapter.adoc]: Suggestion for separate PR (so not blocking this PR):
Here ` are added, but below there are removed.

File: optaplanner-docs/src/main/asciidoc/BenchmarkingAndTweaking/BenchmarkingAndTweaking-chapter.adoc [hunk 2]
```diff
@@ -195,7 +195,7 @@ There are several ways to serialize a solution:
 
 
 [[xStreamSolutionFileIO]]
-==== ``XStreamSolutionFileIO``: Serialize To And From An XML Format
+==== `XStreamSolutionFileIO: serialize to and from an XML format
```
Comment (ge0ffrey) [inline optaplanner-docs/src/main/asciidoc/BenchmarkingAndTweaking/BenchmarkingAndTweaking-chapter.adoc]: Suggestion for separate PR (so not blocking this PR):
Here ` are remove, but above there are added.

**Comment by ge0ffrey** [comment]
Thanks Emily!

File: optaplanner-docs/src/main/asciidoc/BenchmarkingAndTweaking/BenchmarkingAndTweaking-chapter.adoc (hunk 1 above)
Comment (ge0ffrey) [reply optaplanner-docs/src/main/asciidoc/BenchmarkingAndTweaking/BenchmarkingAndTweaking-chapter.adoc]: Note that it wasn't consistent to start out with - which is something we'll want to clean up for consistency. (This PR is fine, nothing wrong with it.)

**Comment by ge0ffrey** [comment]
One thing for future reference for the PR title: the "P" of "OptaPlanner" is a capital (and there is no space in front of it). It's a brand name. Maybe not the best decision we've ever made, but it's like that until someone opens the discussion to change it.

**Comment by emmurphy1** [comment]
Noted, thanks.

**Pull request merged**